conference.js

```diff
@@ -2041,7 +2042,18 @@ export default {
 
         room.on(
             JitsiConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
-            (...args) => APP.store.dispatch(endpointMessageReceived(...args)));
+            (...args) => {
+                APP.store.dispatch(endpointMessageReceived(...args));
+                if (args && args.length >= 2 && args[1].name === ENDPOINT_TEXT_MESSAGE_NAME) {
```

Comment:
can you extract this to objects to make it more readable? like `particcipant` and `data` for example.